bundle/manifests/rhdh.redhat.com_orchestrators.yaml

```diff
@@ -249,6 +249,26 @@ spec:
                               available.
                             type: string
                         type: object
+                      gitlab:
+                        description: Gitlab specific configuration fields that are
+                          injected to the backstage instance to allow the plugin to
+                          communicate with Gitlub.
+                        properties:   
+                          token:
+                            default: GITLAB_TOKEN
+                            description: Key in the secret with name defined in the
+                              'name' field that contains the value of the authentication
+                              token as expected by Gitlab. Required for importing
+                              resource to the catalog, launching software templates
+                              and more. Defaults to 'GITLAB_TOKEN', empty for not
+                              available.
+                            type: string
+                          host:
+                            default: gitlab-host
```

Comment:
Since this is a secret ref, let's be consistent with uppercase? As in `GITLAB_HOST`